app/controllers/pricing_controller.rb

```diff
@@ -8,12 +8,6 @@ def show
   private
 
   def template
-    if Feature.enabled?(:pricing_v3, user: current_user)
-      "pricing/v3/show"
-    elsif Feature.enabled?(:pricing_v2, user: current_user)
-      "pricing/v2/show"
-    else
-      "pricing/v1/show"
-    end
+    "pricing/v3/show"
```

Comment:
We can now remove this entire method and let the `#show` method render `app/views/pricing/show.html.erb`. This means removing all of the v1 and v2 templates/partials and moving the v3 stuff to the "root" level of that directory.

Reply:
I have move the v3 view to root and improve the controller